gradle-plugin/README.md

```diff
@@ -1,16 +1,34 @@
-# Android Product Team's Gradle plugin
+# Mozilla Android gradle-plugin
+The [`org.mozilla.android` gradle plugin, hosted on plugins.gradle.org][plugins], is used to share build code across Mozilla's Android projects like Firefox for Fire TV.
```

Comment:
Below it is written that the newer plugin syntax is not supported, but this link shows it as a valid setup (and seems to imply that it is preferred).  Is that something we have the ability to update?

Reply:
Oops, I seem to have forgotten to include the blurb that the plugin DSL syntax is broken (there is an open issue for it): I'll add that blurb and link to the issue.

Reply:
Oh nevermind, that block is there. We don't have the ability to update the link – plugins.gradle.org just sets up its own syntax. Since we don't have many consumers and we'll probably fix this soon, I'm not too concerned.